zetaclient/chains/base/observer.go

```diff
@@ -546,8 +550,12 @@ func (ob *Observer) PostVoteInbound(
 
 	monitorErrCh := make(chan zetaerrors.ErrTxMonitor, 1)
 
+	// ctxWithTimeout is a context with timeout used for monitoring the vote transaction
+	ctxWithTimeout, _ := zctx.CopyWithTimeout(ctx, context.Background(), MonitoringErrHandlerRoutineTimeout)
```

Comment:
We can just keep in our mind that there could be many 5-minute of dangling timers in the background (hoping not too many), depending how frequent `PostVoteInbound` gets called. No need to change anything now, let's see how it goes.

Reply:
no need to change. The timers will go away after 5 mins.